pr_agent/algo/utils.py

```diff
@@ -375,7 +375,7 @@ def get_user_labels(current_labels: List[str] = None):
 
 def get_max_tokens(model):
     settings = get_settings()
-    max_tokens_model = MAX_TOKENS[model]
+    max_tokens_model = MAX_TOKENS.get(model, 4000)
     if settings.config.max_model_tokens:
         max_tokens_model = min(settings.config.max_model_tokens, max_tokens_model)
```

Comment:
**Suggestion:** The `get_max_tokens` function could be simplified by using the `dict.get()` method's default value feature. This would make the code more readable and concise. [enhancement]